onnxruntime/core/providers/cuda/nn/conv.h

```diff
@@ -125,16 +126,62 @@ struct CudnnConvState {
   CudnnTensor y_tensor;
   CudnnConvolutionDescriptor conv_desc;
 
+  /*
+   Warning: Not thread-safe. Synchronization must be taken care of by the client code.
+   Will return cached memory if the requested size is the same as the cached memory's size
+  */
+  void* AllocMemoryForCuDnnConvResults(AllocatorPtr allocator, size_t size_in_bytes) {
+    // Memory has already been allocated and the requested size matches allocated memory's size
+    if (cached_memory_for_cudnn_results && size_in_bytes == cached_memory_size_in_bytes) {
+      return cached_memory_for_cudnn_results;
```

Comment:
Does it need exact match or would `size_in_bytes <= cached_memory_size_in_bytes` be acceptable as well?

Reply:
Can be <=. I guess we don't have to worry about this given that we need to allocate memory for every Run()

Reply:
This does not exist anymore.